dataland-frontend/src/components/general/AlignedActivitiesDataTable.vue

```diff
@@ -197,6 +222,20 @@ export default defineComponent({
     ];
   },
   methods: {
+    /**
+     * @param value the number to format as a percentage
+     * @returns the new formatted string
+     */
+    valueFormatterForPercentage(value: number | undefined): string {
+      return typeof value !== "undefined" ? `${value}%` : "";
```

Comment:
We are doing percentage conversion in many different places (btw. here the rounding is missing). This should be centralized somewhere.

Reply:
(Btw. the if should more likely be `value !== undefined && value !== null`)

Reply:
The AlignedActivitiesDataTable.vue is being reworked in DALA-2307. Re-Check if issues persist after merge (I.e., do not work on this issue until DALA-2307 has been merged)

Reply:
@stewal19 will look into this